### ISSUE TITLE
Lua module registered to run before or after "AuthenticateRequest" event doesn't get triggered.

### ISSUE THREAD

**Issue by ulasbilgen** [heroiclabs/nakama#78]
## Description
Registering a lua method to Authentication request in server runtime doesn't get triggered. Found out that the event should be registered to ".AuthenticateRequest_Device" event not "AuthenticateRequest"

## Steps to Reproduce
1. Create a simple Lua module, and register it to before or after "AuthenticationRequest" event
2. Login from Unity client to trigger the module. 
3. The event is not fired, so change the event name to ".AuthenticateRequest_Device"
4. Login from Unity client to trigger the module.

## Sample Lua Module
```lua
local nk = require("nakama")
local nx = require("nakamax")

local function TriggerBefore(context,payload)
  nk.logger_info("Triggered BEFORE")
  return payload
end

nk.register_before(TriggerBefore, "AuthenticationRequest")
```

If we change the event name to register, it gets triggered.

```lua
nk.register_before(TriggerBefore, ".AuthenticateRequest_Device")
```

## Expected Result
The event should be triggered, in our sample code the method writes a log into Nakama log file.

## Actual Result
The even is not triggered.

## Context
- [X] Unity

## Your Environment
- Nakama: 0.13.1+4ad68d0
- Cockroach: 1.0.1

**Comment by mofirouz** [comment]
This is fixed in aa0ee02e26e6bba415bedde85885c8ac597f14d7. We are planning to revamp the way we map the Before/After Runtime triggers and this will be revisited to make sure that we have a uniform naming strategy in place.

**Comment by mofirouz** [comment]
This is now done as part of #86 and is merged into master and will be available in 0.14.0+.
